### PR TITLE
improved documentation in GridGenerator

### PR DESCRIPTION
**Pull request by axdn**
Hi I added the Parameters for the parallelogram() function  in GridGenerator.

Andr[es

### CODE REVIEW

File: include/deal.II/grid/grid_generator.h [hunk 1]
```diff
@@ -523,6 +533,7 @@ namespace GridGenerator
    * vertices in the array of <code>corners</code> will no longer refer to the
    * same triangulation.
    *
+   *
```
Comment (jppelteret) [inline include/deal.II/grid/grid_generator.h]:
```suggestion
```

Comment (jppelteret) [reply include/deal.II/grid/grid_generator.h]: I guess this stray addition was accidental.

File: include/deal.II/grid/grid_generator.h [hunk 2]
```diff
@@ -355,6 +355,12 @@ namespace GridGenerator
    * indicators for the surfaces in the y-direction are 2 and 3, and the ones
    * for the z-direction are 5 and 6. The hole boundary has indicator 4.
    *
+   *@param tria the Triangulation to create. It needs to be empty upon
```
Comment (jppelteret) [inline include/deal.II/grid/grid_generator.h]:
```suggestion
   *@param tria The triangulation to be created. It needs to be empty upon
```

File: include/deal.II/grid/grid_generator.h [hunk 3]
```diff
@@ -355,6 +355,12 @@ namespace GridGenerator
...
+   * calling this function.
+   *
+   *@param
```
Comment (jppelteret) [inline include/deal.II/grid/grid_generator.h]:
```suggestion
```

File: include/deal.II/grid/grid_generator.h [hunk 4]
```diff
@@ -355,6 +355,12 @@ namespace GridGenerator
    * indicators for the surfaces in the y-direction are 2 and 3, and the ones
    * for the z-direction are 5 and 6. The hole boundary has indicator 4.
    *
+   *@param tria the Triangulation to create. It needs to be empty upon
+   * calling this function.
+   *
+   *@param  
+   *
```
Comment (jppelteret) [inline include/deal.II/grid/grid_generator.h]:
```suggestion
```

File: include/deal.II/grid/grid_generator.h [hunk 5]
```diff
@@ -355,6 +355,12 @@ namespace GridGenerator
    * indicators for the surfaces in the y-direction are 2 and 3, and the ones
    * for the z-direction are 5 and 6. The hole boundary has indicator 4.
    *
+   *@param tria the Triangulation to create. It needs to be empty upon
+   * calling this function.
+   *
```
Comment (jppelteret) [inline include/deal.II/grid/grid_generator.h]:
```suggestion
```

Comment (jppelteret) [reply include/deal.II/grid/grid_generator.h]: I believe that you accidentally added this and the following two lines.

File: include/deal.II/grid/grid_generator.h [hunk 6]
```diff
@@ -501,7 +507,11 @@ namespace GridGenerator
    * point will be the sum of these two vectors.  Colorizing is done in the
    * same way as in hyper_rectangle().
    *
-   * @note This function is implemented in 2d only.
+   * @note This function is implemented in 2d only. 
+   * 
+   * @param tria Triangulation to create.
```
Comment (jppelteret) [inline include/deal.II/grid/grid_generator.h]:
```suggestion
   * @param tria The triangulation to be created.
```

File: include/deal.II/grid/grid_generator.h [hunk 7]
```diff
@@ -501,7 +507,11 @@ namespace GridGenerator
...
+   * @param corners Adjacent points. 
+   * @param colorize Assign different boundary ids if set to true.
```
Comment (jppelteret) [inline include/deal.II/grid/grid_generator.h]: Would you mind please adding a reference to the glossary entry on colorization, similar to that which is done for the other functions?

File: include/deal.II/grid/grid_generator.h [hunk 8]
```diff
@@ -501,7 +507,11 @@ namespace GridGenerator
    * point will be the sum of these two vectors.  Colorizing is done in the
    * same way as in hyper_rectangle().
    *
-   * @note This function is implemented in 2d only.
+   * @note This function is implemented in 2d only. 
+   * 
+   * @param tria Triangulation to create.
+   * @param corners Adjacent points.
```
Comment (jppelteret) [inline include/deal.II/grid/grid_generator.h]: Could you please elaborate a bit in your description of this parameter?

File: include/deal.II/grid/grid_generator.h (hunk 2 above)
Comment (drwells) [inline include/deal.II/grid/grid_generator.h]: We need a space before `@param`.

File: include/deal.II/grid/grid_generator.h (hunk 3 above)
Comment (drwells) [inline include/deal.II/grid/grid_generator.h]: I think you meant to put something here :)

File: include/deal.II/grid/grid_generator.h (hunk 1 above)
Comment (drwells) [inline include/deal.II/grid/grid_generator.h]: Please remove the blank line.

**Comment by tjhei** [comment]
You will also need to fix indentation using  .``/contrib/utilities/indent``. Track one of us done and we can explain.

**Comment by tjhei** [comment]
Finally, can you add your firstname and lastname to your github account?

File: include/deal.II/grid/grid_generator.h [hunk 9]
```diff
@@ -501,7 +504,14 @@ namespace GridGenerator
    * point will be the sum of these two vectors.  Colorizing is done in the
    * same way as in hyper_rectangle().
    *
-   * @note This function is implemented in 2d only.
+   * @note This function is implemented in 2d only. 
+   * 
+   * @param tria Triangulation to be created.
+   * @param corners Adjacent points. The @p dim
+   * adjacent points are the ones given in the second argument and the fourth
+   * point will be the sum of these two vectors.
```
Comment (drwells) [inline include/deal.II/grid/grid_generator.h]: We should combine this text with the paragraph above (or simply delete that paragraph): there is no need to say the same thing twice.

**Comment by drwells** [comment]
@axdn Let us know if you need help finishing this.

**Comment by drwells** [comment]
@axdn is it all right with you if I move this patch to another PR to finish this work?

**Comment by axdn** [comment]
Hi sure, I got busy with school and I didn't have time for ir!

Best

Andrés
________________________________
De: David Wells <notifications@github.com>
Enviado: viernes, 6 de septiembre de 2019 2:46:00 a. m.
Para: dealii/dealii <dealii@noreply.github.com>
Cc: Galindo Olarte, Andres <galindoo@msu.edu>; Mention <mention@noreply.github.com>
Asunto: Re: [dealii/dealii] improved documentation in GridGenerator (#8488)


@axdn<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_axdn&d=DwMCaQ&c=nE__W8dFE-shTxStwXtp0A&r=sIRNNixVlwyqiDevaofM1Q&m=OKFt-1UTLjlyscZHV9ZWBvBLykbeEaYufTS1pACIeOU&s=6zofNq35O1pohFG8Y8Baf8zNer7j3uzT1WxSzanfFAE&e=> is it all right with you if I move this patch to another PR to finish this work?

**Comment by drwells** [comment]
Superceded by #8827.

**Pull request closed without merging**